tracer/src/Datadog.Trace/Debugger/LiveDebugger.cs

```diff
@@ -280,6 +284,89 @@ internal void UpdateAddedProbeInstrumentations(IReadOnlyList<ProbeDefinition> ad
             }
         }
 
+        private static string[] SafeParseSignature(string signature)
+        {
+            if (string.IsNullOrEmpty(signature))
+            {
+                return null;
```

Comment:
`#nullable disable` makes me sad 😢

Reply:
It will mess with `LiveDebugger` unfortunately. We'll come back to it for sure :)

Reply:
I know, it still makes me sad 😂